File: sdkwork-backend-sdk-typescript/src/types/page-plus-tenant-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusTenantVO } from './plus-tenant-vo';
import type { SortObject } from './sort-object';

export interface PagePlusTenantVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusTenantVO[];
  number?: number;
  numberOfElements?: number;
  first?: boolean;
  last?: boolean;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/page_plus_tenant_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PagePlusTenantVO:
    total_elements: int = None
    total_pages: int = None
    size: int = None
    content: List[PlusTenantVO] = None
    number: int = None
    number_of_elements: int = None
    sort: SortObject = None
    pageable: PageableObject = None
    first: bool = None
    last: bool = None
    empty: bool = None
